samples/flake_id_generator/flake_id_generator_sample.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"github.com/hazelcast/hazelcast-go-client"
+	"strconv"
+)
+
+func main() {
+
+	config := hazelcast.NewHazelcastConfig()
+	config.ClientNetworkConfig().AddAddress("127.0.0.1:5701")
+	client, err := hazelcast.NewHazelcastClientWithConfig(config)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	flakeIdGenerator, _ := client.GetFlakeIdGenerator("generator")
+	id, _ := flakeIdGenerator.NewId()
+	fmt.Println("new id :", strconv.Itoa(int(id)))
```

Comment:
Isnt there an simpler way to print an `int64` in go ?
Usage of Itoa looks more complext than necessary.